conf/application.conf

```diff
@@ -92,6 +92,7 @@ enrolment-config {
 
 pagination-config {
     recursive-page-size = 10000
-    starting-page = 0
-    page-size = 10
-}
+    recursive-starting-page = 0
+    local-page-size = 10
```

Comment:
hmmm....I'd prefer one env variable for this but change the config in app-config-base to 1000 page size - also why is the starting page different?

pagination-config {
     recursive-page-size = 10000
     recursive-starting-page = 0
}